### ISSUE TITLE
Debug mode doesn't work with camera_calibration: true

### ISSUE THREAD

**Issue by sibero80** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#73]
Thank you for work
I have installed this card and Xiaomi-Cloud-Map-Extractor in HASS 0.116.4 through HACS.

The card displays the map correctly, but I cannot draw zones nor select a target point.

Am I doing something wrong with how I setup things?

Here is the only error in the Log file:

```
Logger: frontend.js.latest.202010012
Source: components/system_log/__init__.py:193
First occurred: 12:26:26 AM (145 occurrences)
Last logged: 12:32:55 AM

http://LOCALHASSIP/hacsfiles/lovelace-xiaomi-vacuum-map-card/xiaomi-vacuum-map-card.js:424:58 Uncaught TypeError: this._config.calibration_points is not iterable
```

This is my configuration:

```
camera:
  - platform: xiaomi_cloud_map_extractor
    host: !secret xiaomi_vacuum_host
    token: !secret xiaomi_vacuum_token
    username: !secret xiaomi_cloud_username
    password: !secret xiaomi_cloud_password
    country: "de"
    name: "WalterJr"
    draw:
      - charger
      - path
      - goto_path
      - predicted_path
      - no_go_zones
      - no_mopping_zones
      - vacuum_position
      - virtual_walls
      - zones
    sizes:
      charger_radius: 4
      vacuum_radius: 6.5
    attributes:
      - calibration_points
      - charger
      - goto
      - goto_path
      - goto_predicted_path
      - image
      - is_empty
      - map_name
      - no_go_areas
      - no_mopping_areas
      - obstacles
      - path
      - room_numbers
      - rooms
      - vacuum_position
      - vacuum_room
      - walls
      - zones
    scan_interval:
      seconds: 10
    auto_update: true
```
And this is my  Card configuration:

```
type: 'custom:xiaomi-vacuum-map-card'
entity: vacuum.xiaomi_vacuum
map_camera: camera.walterjr
camera_calibration: true
debug: true
```

Thank you again.

**Comment by PiotrMachowski** [comment]
Which version of a card do you use? Do you use HACS?

Does `camera.walterjr` have `calibration_points` in parameters? If so, can you paste them here?

**Comment by sibero80** [comment]
I use HACS. 
I have not set up calibration points for `camera.walterjr` as it is my underestanding that `Xiaomi-Cloud-Map-Extractor` does not need it, and generatets it's own entities with this information.
In the card configuration I use the `camera_calibration` parameter, that as I underestand, automatically callibrates using the `Xiaomi-Cloud-Map-Extractor` entities.
Here are the calibration points provivided by the `Xiaomi-Cloud-Map-Extractor `entities:

```
access_token: TOKEN
calibration_points: 
- vacuum:
    x: 25500
    'y': 25500
  map:
    x: 126
    'y': 224
- vacuum:
    x: 26500
    'y': 25500
  map:
    x: 146
    'y': 224
- vacuum:
    x: 26500
    'y': 26500
  map:
    x: 146
    'y': 204

charger: 
x: 25646
'y': 25222

goto: null
goto_path: null
goto_predicted_path: null
image: 
size: 146012
offset_y: 313
offset_x: 384
height: 422
scale: 1
rotation: 0
width: 346

is_empty: false
map_name: rubyslite%SOMEID%2F5
no_go_areas: 
no_mopping_areas: 
obstacles: null
path: 
point_length: 0
point_size: 4
angle: 0
path: []

room_numbers: 1, 2, 3, 4, 5, 6, 7, 8
rooms: 
'1':
  x0: 23300
  y0: 20300
  x1: 30000
  y1: 26200
'2':
  x0: 29900
  y0: 24050
  x1: 31950
  y1: 28750
'3':
  x0: 23600
  y0: 25150
  x1: 26950
  y1: 27500
'4':
  x0: 26950
  y0: 25750
  x1: 27800
  y1: 28650
'5':
  x0: 28000
  y0: 26500
  x1: 29850
  y1: 28600
'6':
  x0: 26950
  y0: 28600
  x1: 29850
  y1: 30050
'7':
  x0: 26950
  y0: 30000
  x1: 27700
  y1: 32050
'8':
  x0: 29100
  y0: 30100
  x1: 29850
  y1: 32100

vacuum_position: 
x: 25646
'y': 25398
a: 4294967206

vacuum_room: 3
walls: 
- x0: 26200
  y0: 22150
  x1: 28050
  y1: 22150

zones: null
friendly_name: WalterJr
entity_picture: /api/camera_proxy/camera.walterjr?token=TOKEN
supported_features: 0

```

Am I missinterpreting any `xiaomi-vacuum-map-card` or` Xiaomi-Cloud-Map-Extractor` setup or configuration procedure?

**Comment by PiotrMachowski** [comment]
Ok, I have found a bug in a code. Thanks!

**Comment by sibero80** [comment]
No good sir, thank you!

**Comment by sibero80** [comment]
I see the issue was renamed to be related with Debug Mode, does this mean that I should disable this mode in order to be able to create the zones and point the vacuum to a target?

**Comment by PiotrMachowski** [comment]
Yes, everything should be working without `debug: true`

**Comment by sibero80** [comment]
Indeed it does, thank you.

**Comment by PiotrMachowski** [comment]
I have fixed this problem in v1.2.1
